pimd/pim_vxlan.c

```diff
@@ -253,8 +253,17 @@ static void pim_vxlan_orig_mr_up_del(struct pim_vxlan_sg *vxlan_sg)
 		/* if there are other references register the source
 		 * for nht
 		 */
-		if (up)
-			pim_rpf_update(vxlan_sg->pim, up, NULL, __func__);
+		if (up) {
+			enum pim_rpf_result r;
+
+			r = pim_rpf_update(vxlan_sg->pim, up, NULL, __func__);
+			if (r == PIM_RPF_FAILURE) {
+				if (PIM_DEBUG_VXLAN)
```

Comment:
could save an indent by combining these two into a single condition

Reply:
I thought about that but I believe long term we'll be adding more error handling here.  So I think this is ok at the moment